src/include/processor/operator/persistent/insert_executor.h

```diff
@@ -42,9 +42,11 @@ class RelInsertExecutor {
 public:
     RelInsertExecutor(storage::RelsStatistics& relsStatistics, storage::RelTable* table,
         const DataPos& srcNodePos, const DataPos& dstNodePos,
+        std::vector<DataPos> lhsVectorPositions,
```

Comment:
I would recommend adding a comment in insert_executor.h to give a very simple example on what lhs and rhs are. It will be much more easier to understand.